### ISSUE TITLE
Unknown TypeofTypeAnnotation

### ISSUE THREAD

**Issue by Kerumen** [brigand/babel-plugin-flow-react-proptypes#25]
I have the following error:

```
Encountered an unknown node in the type definition. 
Node: {"type":"TypeofTypeAnnotation","start":719,"end":745,"loc":{"start":{"line":27,"column":8},"end":{"line":27,"column":34}},"argument":{"type":"GenericTypeAnnotation","start":726,"end":745,"loc":{"start":{"line":27,"column":15},"end":{"line":27,"column":34}},"typeParameters":null,"id":{"type":"QualifiedTypeIdentifier","start":726,"end":745,"loc":{"start":{"line":27,"column":15},"end":{"line":27,"column":34}},"qualification":{"type":"Identifier","start":726,"end":737,"loc":{"start":{"line":27,"column":15},"end":{"line":27,"column":26}},"name":"ActionTypes"},"id":{"type":"Identifier","start":738,"end":745,"loc":{"start":{"line":27,"column":27},"end":{"line":27,"column":34}},"name":"JUMP_TO"}}}}
```

Here: [NavigationTabsReducer.js#L27](https://github.com/facebook/react-native/blob/0.26-stable/Libraries/NavigationExperimental/Reducer/NavigationTabsReducer.js#L27)

**Comment by brigand** [comment]
Oh wow, didn't even know this existed. I think I'm just going it mark it as PropTypes.any because it looks really complex and bug prone to do the smart thing here.

Thanks for the report!

**Comment by brigand** [comment]
Published as 0.7.4.
